esi_leap/resource_objects/ironic_node.py

```diff
@@ -101,6 +113,10 @@ def resource_admin_project_id(self):
         return self._get_node().owner
 
     def _get_node(self, resource_list=None):
-        if not self._node:
-            self._node = ironic.get_node(self._uuid, resource_list)
+        try:
+            if not self._node:
+                self._node = ironic.get_node(self._uuid, resource_list)
+        except exceptions.NotFound:
+            self._node = FakeIronicNode()
```

Comment:
Let's not call it "FakeIronicNode" here - maybe "UnknownIronicNode"?

Reply:
We'll also want to make sure that _get_node has a unit test that travels down the UnknownIronicNode path.

Reply:
Okay!